tests/end-to-end/version.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+phpunit --version
+--FILE--
+<?php
+$_SERVER['argv'][1] = '--no-configuration';
+$_SERVER['argv'][2] = '--version';
+
+require __DIR__ . '/../bootstrap.php';
+
+PHPUnit\TextUI\Command::main();
+?>
+--EXPECTF--
+PHPUnit %s by Sebastian Bergmann and contributors.
```

Comment:
I'm unaware of it, is there a possibility to denote the end of the expected output?

Currently, another empty line is rendered after the version information, and the test also passes with the following diff applied

```diff
diff --git a/tests/end-to-end/version.phpt b/tests/end-to-end/version.phpt
index bdd38596c..af9ee63cc 100644
--- a/tests/end-to-end/version.phpt
+++ b/tests/end-to-end/version.phpt
@@ -11,3 +11,7 @@ PHPUnit\TextUI\Command::main();
 ?>
 --EXPECTF--
 PHPUnit %s by Sebastian Bergmann and contributors.
+
+
+
+
```

Reply:
Ha, the section content is trimmed, see [`PhptTestCase::assertPhptExpectation`](https://github.com/sebastianbergmann/phpunit/blob/7.5.16/src/Runner/PhptTestCase.php#L305).